src/x509.c

```diff
@@ -8013,23 +8037,39 @@ void wolfSSL_X509_CRL_free(WOLFSSL_X509_CRL *crl)
 #endif /* HAVE_CRL && (OPENSSL_EXTRA || WOLFSSL_WPAS_SMALL) */
 
 #ifdef OPENSSL_EXTRA
-#ifndef NO_WOLFSSL_STUB
 WOLFSSL_ASN1_TIME* wolfSSL_X509_CRL_get_lastUpdate(WOLFSSL_X509_CRL* crl)
 {
-    (void)crl;
-    WOLFSSL_STUB("X509_CRL_get_lastUpdate");
-    return 0;
+    if ((crl != NULL) && (crl->crlList != NULL) &&
```

Comment:
I think it's preferable to do this copying logic for lastUpdate and nextUpdate in InitCRL_Entry. Then wolfSSL_X509_CRL_get_lastUpdate and wolfSSL_X509_CRL_get_nextUpdate just return the appropriate member of the struct. There's also a function DupCRL_Entry that hasn't been updated to duplicate the new fields you've added.